utils/hct/hctdb.py

```diff
@@ -1175,6 +1175,37 @@ def populate_llvm_instructions(self):
         self.add_llvm_instr(
             "OTHER", 53, "VAArg", "VAArgInst", "vaarg instruction", "", []
         )
+
+        self.add_llvm_instr(
```

Comment:
What does it mean to `add_llvm_instr` in this case? I don't think it means anything about validity because i've seen these instructions emitted by DXC before?

Does this means there is a way to print out all the llvm instructions DXC supports via querying the `DirectXShaderCompiler.utils.hct.hctdb_instrhelp` package? 

I'm curious for my own reasons because I want to build a  [Set difference](https://en.wikipedia.org/wiki/Set_difference) of DXC to upstream llvm and know which instructions need to be legalized.